packages/zowe-explorer-api/__mocks__/vscode.ts

```diff
@@ -921,16 +921,6 @@ export namespace workspace {
         return Disposable;
     }
 
-    /**
-     * ~~The folder that is open in the editor. `undefined` when no folder
-     * has been opened.~~
-     *
-     * @deprecated Use [`workspaceFolders`](#workspace.workspaceFolders) instead.
-     *
-     * @readonly
-     */
-    export let rootPath: string | undefined;
-
```

Comment:
Was this removed from VS Code types? I'm curious to know why this was deleted from mocks 😋

Reply:
Good catch, thanks for updating that!